src/studiolibrary/librarywidget.py

```diff
@@ -910,8 +919,19 @@ def updateItems(self):
             return
 
         # Create a query using the sidebar and search widgets
-        queries = self.sidebarQuery(), self.searchQuery()
-
+        queries = [self.sidebarQuery(), self.searchQuery()]
+        
+        #we do not want folder in the view
+        if not self.areFoldersVisible():
```

Comment:
Thanks! I have another solution which I will be adding soon, however, this is good for now. 👍